docs/src/main/sphinx/functions/math.md

```diff
@@ -226,12 +226,34 @@ Returns the value of `string` interpreted as a base-`radix` number.
 Returns the base-`radix` representation of `x`.
 :::
 
+## Geometrical functions
+
+:::{function} cosine_distance(array(double), array(double)) -> double
+Calculates the cosine distance of two double arrays:
+
+```sql
+SELECT cosine_distance(ARRAY[1.0, 2.0], ARRAY[3.0, 4.0]);
+-- 0.01613008990009257
+```
+:::
+
+:::{function} cosine_similarity(array(double), array(double)) -> double
+Calculates the cosine similarity of two double arrays:
+
+```sql
+SELECT cosine_similarity(ARRAY[1.0, 2.0], ARRAY[3.0, 4.0]);
+-- 0.9838699100999074
+```
+:::
+
 ## Statistical functions
 
+(statistical-cosine-similarity)=
 :::{function} cosine_similarity(x, y) -> double
+:no-index:
 Returns the cosine similarity between the sparse vectors `x` and `y`:
 
-```
+```sql
```

Comment:
Having this kinda begs the questions if we should add cosine distance for sparse vectors as well .. should be the same 1-similarity from what I know

Reply:
wdyt @dain ?